bundles/org.openhab.binding.somneo/src/main/java/org/openhab/binding/somneo/internal/SomneoBindingConstants.java

```diff
@@ -58,6 +58,21 @@ public class SomneoBindingConstants {
     public static final String CHANNEL_SUNSET_REMAINING_TIME = "sunset#remainingTime";
     public static final String CHANNEL_SUNSET_SWITCH = "sunset#switch";
     public static final String CHANNEL_SUNSET_VOLUME = "sunset#volume";
+    public static final String CHANNEL_ALARM_SNOOZE = "alarm#snooze";
```

Comment:
Thank you for the review. What exactly do you mean by separate? Rename the group id alarm because it is too similar to the other alarm groups ids with an index (alarm{index})?

Reply:
@clinique - can you help clarifying this?

Reply:
@clinique - gentle ping.

Reply:
`alarm` is the group name, while `snooze` is the channel itself. You could directly target it using `channelUID.getIdWithoutGroup();` in your handler.

Reply:
@0x4d4d - did you see @clinique's reply?

Reply:
@jlaur Sorry, I had commented but it had the pending label and don't saw it.

Reply:
I don't see any issues with the current approach that should block this PR longer.

@clinique - considering @0x4d4d's latest comment, please continue and conclude the discussion here and do a follow-up PR if there is anything that can be improved without introducing unnecessary complexity or redundancy. I have seen similar comments from you in other PR's, so I think it would be nice to reach a conclusion/consensus.

You are also welcome to have a look at #14376 where I have something similar and couldn't find a better solution:

https://github.com/openhab/openhab-addons/pull/14376/files#diff-002ecf5a3879dda71cf4fcb82dbfef3f550c0f1655cfb16134fa3e4f3b431d69R42-R54